app/Models/Winner.php

```diff
@@ -12,7 +12,7 @@ class Winner extends Model
      * @var array
      */
     protected $fillable = [
-        'rank', 'description'
+        'candidate_id', 'rank', 'description'
```

Comment:
I think it's okay. We were just filling that field with user input manually before (plus, the route is blocked to admin users so I'm not too scared). :ghost: